docs/misc/VolumeNames.md

```diff
@@ -1,12 +1,19 @@
 # Short Volume names
-Short Volume Name can contain up to a 100 of alphanumeric characters, underscores and dots.
-Examples: `MyVolume_fast.12` or `_12SuperStorage`.
+Short Volume Name can contain up to a 100 characters.
+All characters are allowed, but the last '@' will be interpreted as a separator
+between volume and datastore name (see below).
+As an exception, we do not allow volume names ending with '-NNNNNNN', where
+N is a digit (this limitation is to prevent conflicts with ESXi deltadisks used
+during VM snapshots). Examples: `MyVolume_fast.12`,  `_12SuperStorage`,
+`super-volume-200` are all valid names, while `myVolume-000012` is not valid.
+
 
```

Comment:
Ah, but it only applies to local volumes. Docker leaves volume name constraints to drivers. For example, spaces in vmdk volume name work just fine with this PR